src/bootstrap/src/core/config/flags.rs

```diff
@@ -388,6 +388,9 @@ pub enum Subcommand {
         /// enable this to generate a Rustfix coverage file, which is saved in
         /// `/<build_base>/rustfix_missing_coverage.txt`
         rustfix_coverage: bool,
+        #[arg(long)]
+        /// don't capture stdout/stderr of tests
+        no_capture: bool,
     },
```

Comment:
Suggestion: since this is adding a new flag, can you add a change tracker INFO-level entry?

Reply:
EDIT: the warning-level u added is more suitable indeed.